src/field/pokemon.ts

```diff
@@ -3393,6 +3438,16 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     return maxForms.includes(this.getFormKey());
   }
 
+  isMega(): boolean {
+    const megaForms = [
+      SpeciesFormKey.MEGA,
+      SpeciesFormKey.MEGA_X,
+      SpeciesFormKey.MEGA_Y,
+      SpeciesFormKey.PRIMAL,
+    ] as string[];
+    return megaForms.includes(this.getFormKey());
```

Comment:
Q: Can't you define the array as `string[]` to begin with?
```suggestion
    const megaForms: string[] = [
      SpeciesFormKey.MEGA,
      SpeciesFormKey.MEGA_X,
      SpeciesFormKey.MEGA_Y,
      SpeciesFormKey.PRIMAL,
    ];
    return megaForms.includes(this.getFormKey());
```

Reply:
Probably, this was copy-pasted. Will check.